language-support/java/bindings/BUILD.bazel

```diff
@@ -120,6 +120,9 @@ da_scala_test_suite(
         "@maven//:org_scalatestplus_scalacheck_1_15",
         "@maven//:org_scala_lang_modules_scala_collection_compat",
     ],
+    scalacopts = [
+        "-P:silencer:globalFilters=deprecated",  # tests need to check the behavior of deprecated fields
+    ],
```

Comment:
The filter should match the names of the deprecated fields, at least. Unless the error message doesn't contain the field name.

Reply:
Addressed by https://github.com/digital-asset/daml/pull/11839/commits/dd39b08c819394ef30911038e4f76dabfc2a70e9